### ISSUE TITLE
Latest build throws error when compiling cee_wks, codeman.cpp

### ISSUE THREAD

**Issue by cydhaselton** [dotnet/coreclr#12982]
When building latest version of repo (last commit https://github.com/dotnet/coreclr/commit/094f1fd729be3483a40336daad5a81e71bf5179f), the folllowing error is thrown

```
In file included from /build/core/coreclr/src/vm/codeman.cpp:9:
In file included from /build/core/coreclr/src/vm/common.h:91:
/build/core/coreclr/src/inc/volatile.h:128:47: error: cannot initialize a parameter of type 'int *' with an rvalue of type 'volatile int *'
__atomic_load((T volatile const *)pt, &val, __ATOMIC_ACQUIRE);
```

**Comment by jkotas** [comment]
cc @sdmaclea

**Comment by cydhaselton** [comment]
@jkotas @sdmaclea I have a PR in for this issue, made after submitting it #12986

**Comment by cydhaselton** [comment]
Closed by #13055
